app/models/user.rb

```diff
@@ -9,4 +9,6 @@ class User < ApplicationRecord
     validates_presence_of :password, require: true
 
     has_secure_password
+
+    enum role: {default: 0, merchant: 1, admin: 2}
```

Comment:
This might lead to a merge conflict.
There might be a migrations conflict since I also made one for US13 and 14